.gitattributes

```diff
@@ -1,6 +1,13 @@
+# Default behaviour, for if core.autocrlf isn't set
+* text=auto
+
+# Binary files
 boot/ocamlc binary
 boot/ocamllex binary
 boot/ocamldep binary
+*.gif
+*.png
+*.tfm
```

Comment:
Does this just come from a default set of recommended application-agnostic patterns, or is there an actual use case for manipulating pictures inside ocaml/ocaml?